cmd/argocd-application-controller/main.go

```diff
@@ -46,6 +46,7 @@ func newCommand() *cobra.Command {
 		logLevel                 string
 		glogLevel                int
 		metricsPort              int
+		kubectlParallelismLimit  int64
```

Comment:
why `int64`?

Reply:
Eventual value is used as a parameter for `semaphore.NewWeighted` which is `int64`